node/clients/index.ts

```diff
@@ -13,16 +13,23 @@ import StorefrontPermissions from './storefrontPermissions'
 import VtexId from './vtexId'
 
 export const getTokenToHeader = (ctx: IOContext) => {
-  const token =
-    ctx.storeUserAuthToken ?? ctx.adminUserAuthToken ?? ctx.authToken
+  const adminToken = ctx.adminUserAuthToken ?? ctx.authToken
+  const userToken = ctx.storeUserAuthToken ?? null
```

Comment:
A nullish returning `null`? Can this operation be removed?

Reply:
Removed it